### PR TITLE
Autocomm grad pathway

### PR DESCRIPTION
**Pull request by GabyRangelB**
# Pull Request

## Summary & Motivation

[//]: # "When merged, this pull request will..."

## Self-review

- [x] Update **due date**, **assignee**, and **priority** on our
      [TEAMster Asana Project](https://app.asana.com/0/1205971774138578/1205971926225838)

  _If this is a same-day request, please flag that in Slack_

- [x] <kbd>Format</kbd> has been run on all modified files

## Troubleshooting

- [SqlFluff Rules Reference](https://docs.sqlfluff.com/en/stable/rules.html)
- [Trunk](https://teamschools.github.io/teamster/CONTRIBUTING/#trunk)
- [dbt](https://teamschools.github.io/teamster/CONTRIBUTING/#dbt-cloud_1)

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1206789744010862

### CODE REVIEW

File: src/dbt/kipptaf/models/extracts/powerschool/rpt_powerschool__autocomm_students.sql [hunk 1]
```diff
@@ -1,3 +1,24 @@
...
+            s_nj_stu_x__graduation_pathway_math,
+            s_nj_stu_x__graduation_pathway_ela,
+        from
+            grad_path pivot (
+                max(final_grad_path) for discipline in (
+                    'Math' as `s_nj_stu_x__graduation_pathway_math`,
+                    'ELA' as `s_nj_stu_x__graduation_pathway_ela`
+                )
+            )
+    )
```
Comment (cbini) [inline src/dbt/kipptaf/models/extracts/powerschool/rpt_powerschool__autocomm_students.sql:20]: @GabyRangelB here's an example of using the `pivot` operator

**Review by cbini — APPROVED** [review]

File: src/dbt/kipptaf/models/extracts/powerschool/rpt_powerschool__autocomm_students.sql (hunk 1 above)
Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/powerschool/rpt_powerschool__autocomm_students.sql:20]: I tried doing that, but I was getting double rows for students when the pathway letters were not both the same.

Comment (GabyRangelB) [reply src/dbt/kipptaf/models/extracts/powerschool/rpt_powerschool__autocomm_students.sql:20]: So, in other words, I was getting one line for Math where math was filled but ELA was blank, and then one line for ELA where ELA was filed and math was blank.

Comment (cbini) [reply src/dbt/kipptaf/models/extracts/powerschool/rpt_powerschool__autocomm_students.sql:20]: yeah I had to split it up into 2 CTEs to make it pivot correctly. not entirely sure why that happens sometimes

**Pull request merged**